### ISSUE TITLE
Number of training steps for results from paper

### ISSUE THREAD

**Issue by moraval** [bgshih/crnn#7]
Hi,

I was wondering for how many training steps did you train your model to gain the accuracy from the paper. You mentioned that it takes about 50 hours to train the model (on Tesla K40). I wanted to replicate the results, but I stumbled on much longer training time (about 260 hours), that would be required for completing the 2000000 iterations (maxIterations) that are used in config.lua. I'm using Tesla K20, but I don't think the difference should be so big.

**Comment by bgshih** [comment]
@moraval The training should be stopped before reaching the max number of iterations. In practice, The number of training iterations is typically 200-500k.
